### PR TITLE
Code review for MathGame

### PR DESCRIPTION
**Pull request by SinghxRaj**

### CODE REVIEW

File: SinghxRaj.MathGame/Program.cs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿// See https://aka.ms/new-console-template for more information
```
Comment (TheCSharpAcademy) [inline SinghxRaj.MathGame/Program.cs:1]: Please remove comment

File: SinghxRaj.MathGame/Option.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (TheCSharpAcademy) [inline SinghxRaj.MathGame/Option.cs:5]: Please remove unused usings

File: SinghxRaj.MathGame/MathGameHistory.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿namespace SinghxRaj.MathGame.MathGame;
```
Comment (TheCSharpAcademy) [inline SinghxRaj.MathGame/MathGameHistory.cs:1]: please add space between namespace and class name

File: SinghxRaj.MathGame/Program.cs [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+using SinghxRaj.MathGame;
+
```
Comment (TheCSharpAcademy) [inline SinghxRaj.MathGame/Program.cs:3]: Please reinstate deleted project folders from repository

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: SinghxRaj.MathGame/Program.cs (hunk 1 above)
Comment (SinghxRaj) [reply SinghxRaj.MathGame/Program.cs]: Removed the comments

File: SinghxRaj.MathGame/MathGameHistory.cs (hunk 3 above)
Comment (SinghxRaj) [reply SinghxRaj.MathGame/MathGameHistory.cs:1]: Added space between the namespace and class name. I'll be make sure to keep doing this moving forward.

File: SinghxRaj.MathGame/Program.cs (hunk 4 above)
Comment (SinghxRaj) [reply SinghxRaj.MathGame/Program.cs:2]: I ended up cherry picking the commit when I first forked the repo and merged it into HEAD.

File: SinghxRaj.MathGame/Option.cs (hunk 2 above)
Comment (SinghxRaj) [reply SinghxRaj.MathGame/Option.cs]: Removed all the unnecessary using directives.

**Pull request merged**